docs/src/main/asciidoc/deploying-to-openshift.adoc

```diff
@@ -42,7 +42,7 @@ Quarkus offers the ability to automatically generate OpenShift resources based o
 The OpenShift extension is actually a wrapper extension that brings together the link:deploying-to-kubernetes[kubernetes] and link:container-image#s2i[container-image-s2i]
 extensions with sensible defaults so that it's easier for the user to get started with Quarkus on OpenShift.
 
-When we added the `openshift` extension to the command line invocation above, the following dependency was added to the `pom.xml`
+When we added the `OpenShift` extension to the command line invocation above, the following dependency was added to the `pom.xml`
```

Comment:
We usually keep the lowercase extension name when in backticks.

Reply:
I thought this only applies when the full artefact is in use as `quarkus-openshift`  extension.

Reply:
I would either remove the backticks and have it properly cased or keep the backticks and keep it lowercase.

Reply:
Great - here is the followup PR https://github.com/quarkusio/quarkus/pull/8046. Good to know this :-)